Unable to create domain in nucleus distribution

During build,  `java.lang.ClassNotFoundException: jakarta.xml.bind.JAXBContext` is thrown when creating the default domain for the nucleus distribution.

```
[INFO] --- glassfishbuild-maven-plugin:3.2.27:exec (create-domain) @ nucleus-new ---
[INFO] executable: /opt/glassfish/nucleus/distributions/nucleus/target/stage/nucleus/bin/nadmin
[INFO] commandLineArgs: --user admin create-domain --nopassword --savelogin=true --checkports=false --adminport 4848 --instanceport 8080 --keytooloptions CN=localhost domain1
[INFO] [exec] Exception in thread "main" java.lang.NoClassDefFoundError: jakarta/xml/bind/JAXBContext
[INFO] [exec]   at com.sun.enterprise.admin.servermgmt.template.TemplateInfoHolder.parse(TemplateInfoHolder.java:78)
[INFO] [exec]   at com.sun.enterprise.admin.servermgmt.template.TemplateInfoHolder.<init>(TemplateInfoHolder.java:48)
[INFO] [exec]   at com.sun.enterprise.admin.servermgmt.domain.DomainBuilder.initialize(DomainBuilder.java:120)
[INFO] [exec]   at com.sun.enterprise.admin.servermgmt.domain.DomainBuilder.<init>(DomainBuilder.java:85)
[INFO] [exec]   at com.sun.enterprise.admin.servermgmt.cli.CreateDomainCommand.createTheDomain(CreateDomainCommand.java:472)
[INFO] [exec]   at com.sun.enterprise.admin.servermgmt.cli.CreateDomainCommand.executeCommand(CreateDomainCommand.java:308)
[INFO] [exec]   at com.sun.enterprise.admin.cli.CLICommand.execute(CLICommand.java:298)
[INFO] [exec]   at com.sun.enterprise.admin.cli.AdminMain.executeCommand(AdminMain.java:348)
[INFO] [exec]   at com.sun.enterprise.admin.cli.AdminMain.doMain(AdminMain.java:282)
[INFO] [exec]   at com.sun.enterprise.admin.cli.AdminMain.main(AdminMain.java:184)
[INFO] [exec] Caused by: java.lang.ClassNotFoundException: jakarta.xml.bind.JAXBContext
[INFO] [exec]   at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
[INFO] [exec]   at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
[INFO] [exec]   at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
[INFO] [exec]   ... 10 more
[INFO] [exec] Result: 1
[INFO]
```

admin-cli.jar has jakarta.xml.bind-api.jar in MANIFEST.MF but is not included in modules.

--- a/nucleus/distributions/nucleus/src/main/assembly/nucleus-new.xml
+++ b/nucleus/distributions/nucleus/src/main/assembly/nucleus-new.xml
@@ -145,7 +145,6 @@
                 <exclude>felix.jar</exclude>
                 <exclude>nucleus-domain.jar</exclude>
                 <exclude>cluster-cli.jar</exclude>
-                <exclude>jakarta.xml.bind-api.jar</exclude>
             </excludes>
             <outputDirectory>${install.dir.name}/modules</outputDirectory>
         </fileSet>

